modules/client-configuration/pages/clients-oracle.adoc

```diff
@@ -0,0 +1,127 @@
+[[clients-oracle]]
+= Registering {oracle} Clients
+
+This section contains information about registering traditional and Salt clients running {oracle} operating systems.
+
+[IMPORTANT]
+====
+You are responsible for arranging access to {oracle} base media repositories and {oracle} installation media.
+====
+
+ifeval::[{suma-content} == true]
+
+endif::[]
+
+
+[IMPORTANT]
+====
+Traditional clients are not available on {oracle}{nbsp}8.
+{oracle}{nbsp}8 clients are only supported as Salt clients.
+====
+
+
+== Server Requirements
+
+Before you begin, check that your {productname} Server meets the requirements at xref:installation:hardware-requirements.adoc[].
+
+Taskomatic uses one CPU core, and requires at least 3072{nbsp}MB of RAM.
+To ensure that taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+
+----
+taskomatic.java.maxmemory=3072
+----
+
+Restart Taskomatic:
+----
+systemctl restart taskomatic
+----
+
+
+
+== Channel and Repository Management
+
+
+
+
+ifeval::[{suma-content} == true]
+.Procedure: Adding Client Tools Channels
+. On the {productname} Server, add the appropriate {oracle} channels:
++
+* For {oracle} 6:
++
+From the {webui}, add [systemitem]``Oracle Linux 6 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux6-x86_64``.
++
+* For {oracle} 7:
++
+From the {webui}, add [systemitem]``Oracle Linux 7 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux7-x86_64``.
++
+* For {oracle} 8:
++
+From the {webui}, add [systemitem]``Oracle Linux 8 x86_64``.
++
+From the command prompt, add [systemitem]``oraclelinux8-x86_64``.
+. Synchronize the {productname} Server with the {SCC}.
+You can do this using the {webui}, or by running [command]``mgr-sync`` at the command prompt.
+. Add the new channel to your activation key.
+endif::[]
+
+
+ifeval::[{uyuni-content} == true]
+[IMPORTANT]
+====
+For {oracle} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
+You will require packages from both channels.
+If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+====
```

Comment:
@paususe not really if we provide this via sumatoolbox, right?

Reply:
> @paususe not really if we provide this via sumatoolbox, right?

Right. Maybe it could be rewritten like this

"
For {oracle} 8 clients, both the ``BaseOS`` and ``Appstream`` channels need to be selected, as you will require packages from both channels.
If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
"